README.md

```diff
@@ -1,113 +1,18 @@
-<p align="center">
-  <img width="80" height="80" src="https://raw.github.com/mongodb/mongocli/master/mongocli.svg" alt="MongoDB CLI Logo">
-</p>
-
+## MongoDB Atlas CLI
 
 ![GO tests](https://github.com/mongodb/mongocli/workflows/GO%20tests/badge.svg)
 ![golangci-lint](https://github.com/mongodb/mongocli/workflows/golangci-lint/badge.svg)
 
-The MongoDB CLI is a modern command line interface that enables you to manage your MongoDB services from the terminal.
-
-![mongocli-atlas-quickstart](https://user-images.githubusercontent.com/461027/126986233-0dd5c82a-2c75-4887-ab66-eb018c59e093.gif)
+The MongoDB Atlas CLI repository contains the following modern command line interfaces:
+- [AtlasCLI](atlascli.md)
+- [MongoCLI](mongocli.md)
 
 Use simple, one-line commands to interact with MongoDB Atlas, Cloud Manager, or Ops Manager, and to automate management tasks for your deployments.
```

Comment:
could remove "to". "…, and automate management tasks for your deployments."

Reply:
updated